azurerm/resource_arm_container_group_test.go

```diff
@@ -139,6 +140,7 @@ func TestAccAzureRMContainerGroup_windowsComplete(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, "container.0.environment_variables.foo", "bar"),
 					resource.TestCheckResourceAttr(resourceName, "container.0.environment_variables.foo1", "bar1"),
 					resource.TestCheckResourceAttr(resourceName, "os_type", "Windows"),
+					resource.TestCheckResourceAttr(resourceName, "restart_policy", "OnFailure"),
```

Comment:
can we make this `Never` to be able to test that setting is valid? (since `Always` and `OnFailure` are tested above, implicitly and explicitly respectively)

Reply:
Good catch.